command/import.go

```diff
@@ -113,11 +122,27 @@ func runImport(root string, options ForceDeployOptions, displayOptions *deployOu
 		ErrorAndExit(err.Error())
 	}
 
-	err = deploy(force, files, &options, displayOptions)
-	if err == nil && displayOptions.reportFormat == "text" && !displayOptions.quiet {
-		fmt.Printf("Imported from %s\n", root)
-	}
-	if err != nil && (!errors.Is(err, testFailureError) || displayOptions.errorOnTestFailure) {
-		ErrorAndExit(err.Error())
+	var deployments sync.WaitGroup
+	status := deployStatus{aborted: false}
+
+	for _, f := range manager.getAllForce() {
+		if status.isAborted() {
+			break
+		}
+		current := f
+		deployments.Add(1)
+		go func() {
+			defer deployments.Done()
+			err := deployWith(current, &status, files, &options, displayOptions)
```

Comment:
I think we'll need to return the results so we can combine them.  We'll get invalid output, especially with `--reporttype junit`, if we let each goroutine write it its own results.